web/ui-app-v2/src/screens/MyComplaints/components/Complaint/index.js

```diff
@@ -1,81 +1,56 @@
       break;
-    case "CLOSE":
-      style = {
-        color: "#55970a",
+    case "CLOSED":
+      statusObj.style = {
+        color: "#5385a6",
       };
+      statusObj.message = `Complaint resolved. Please rate`;
       break;
     default:
-      style = {
+      statusObj.style = {
         color: "#484848",
       };
+      statusObj.message = `Complaint Re-assigned to ${assignee}`;
   }
-  return style;
+  return statusObj;
 };
 
 const Complaint = ({ index, item, history, onClick }) => {
   return (
     <div id={"complaint-" + index} className="complaints-card-main-cont">
       <Card
-        card={{ padding: 0 }}
         className="complaint-card"
         textChildren={
           <div className="complaint-card-wrapper">
             <div className="complaint-header-cont">
               <span className="complaint-header text-bold dark-color">{item.header}</span>
-              <FlatButton
-                onClick={(e) => {
-                  if (item.status == "ASSIGNED") {
-                    history.push("/complaint-details?status=assigned");
-                  } else if (item.status == "REJECTED") {
-                    history.push("/complaint-details?status=rejected");
-                  }
-                }}
-                className="complaint-track-button"
-                backgroundColor="transparent"
-                label={"Track"}
-                style={{
-                  border: "1px solid #f5a623",
-                  height: "auto",
-                  lineHeight: "auto",
-                  padding: "4px 8px",
-                  minWidth: "inherit",
-                }}
-                labelStyle={{
-                  color: "#f5a623",
-                  padding: 0,
-                  letterSpacing: "0.3px",
-                  display: "inline-block",
-                  height: "14px",
-                  lineHeight: "14px",
-                }}
-                hoverColor="none"
-              />
+              <span className="complaint-status-text text-bold" style={getStatusAndChangeColor(item.status).style}>{` ${item.status}`}</span>
             </div>
-            <div className="complaint-address-cont">
-              <Icon action="maps" name="place" />
-              <span className="complaint-address">{item.address}</span>
+            <div className="complaint-date-cont">
+              <Icon action="action" name="date-range" />
+              <span className="complaint-date">{item.date}</span>
             </div>
-            <div className="complaint-status-cont">
-              <span className="complaint-status-text dark-color text-bold">
-                Status :
-                <span className="text-bold" style={getStatusAndChangeColor(item.status)}>{` ${item.status}`}</span>
-              </span>
+            <div className="complaint-number-cont">
+              <span className="complaint-number complaint-date">Complaint No {item.complaintNo}</span>
```

Comment:
Please try to change all text nodes to Label Components, for easier translation.

For example
 <span className="text-bold" style={getStatusAndChangeColor(item.status)}>{` ${item.status}`}</span>
can be changed to
// might have to do display : inline-block
 <Label  bold={true} labelContainerStyle={getStatusAndChangeColor(item.status)} label={item.status}  />